src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/Unicode/UnicodeCategoryTheory.cs

```diff
@@ -1,77 +1,75 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics;
+using System.Globalization;
+using System.Threading;
+
 namespace System.Text.RegularExpressions.Symbolic.Unicode
 {
-    internal sealed class UnicodeCategoryTheory<TPredicate> where TPredicate : class
+    /// <summary>Utility class providing singleton <see cref="BDD"/>s for evaluating whether a character is part of a particular Unicode category.</summary>
+    internal static class UnicodeCategoryConditions
```

Comment:
You could rename the file accordingly.

Reply:
I plan to subsequently. It can be hard to follow diffs in my experience when there are lots of changes in a file and it's renamed.